DigitalLearningSolutions.Web/Controllers/TrackingSystem/CentreConfigurationController.cs

```diff
@@ -116,5 +119,100 @@ public IActionResult EditCentreWebsiteDetails(EditCentreWebsiteDetailsViewModel
 
             return RedirectToAction("Index");
         }
+
+        [HttpGet]
+        [Route("RegistrationPrompts/{promptNumber}/Edit")]
+        public IActionResult EditRegistrationPrompt(int promptNumber)
+        {
+            var centreId = User.GetCentreId();
+
+            var customPrompt = customPromptsService.GetCustomPromptsForCentreByCentreId(centreId).CustomPrompts
+                .Single(cp => cp.CustomPromptNumber == promptNumber);
+
+            return View(new EditRegistrationPromptViewModel(customPrompt));
+        }
+
+        [HttpPost]
+        [Route("RegistrationPrompts/{promptNumber}/Edit")]
+        public IActionResult EditRegistrationPrompt(EditRegistrationPromptViewModel model, string action)
+        {
+            if (action.StartsWith("delete") && int.TryParse(action.Remove(0, 6), out var index))

```

Comment:
I'm finding this if statement a little bit hard to read - could we pull it out or simplify it in some way? :)